src/modules/shared/components/ReadingRoomCardList/ReadingRoomCardList.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, useEffect, useState } from 'react';
+
+import ReadingRoomCard from '../ReadingRoomCard/ReadingRoomCard';
+
+import styles from './ReadingRoomCardList.module.scss';
+import { ReadingRoomCardListProps } from './ReadingRoomCardList.types';
+
+const defaultProps = {
```

Comment:
In functional components worden deze meestal gwn binnen de function param als default waarde gezet:
`({ items = [], limit = true }) => {}`

Reply:
https://github.com/viaacode/hetarchief-client/pull/44#issuecomment-1009934473
Aangezien het hier over een lege array en een boolean gaat hoeft dat niet perse in een aparte const.
Als er bv een een invulling zou zijn voor die array kan je dat wel doen.